packages/@aws-cdk/cdk/test/cloudformation/test.fn.ts

```diff
@@ -53,7 +53,7 @@ export = nodeunit.testCase({
         fc.property(
           fc.string(), fc.array(nonEmptyString, 1, 3), tokenish, fc.array(nonEmptyString, 1, 3),
           (delimiter, prefix, obj, suffix) =>
-            _.isEqual(resolve(new fn.FnJoin(delimiter, [...prefix, obj, ...suffix])),
+            _.isEqual(resolve(Fn.join(delimiter, [...prefix, obj as any, ...suffix])),
```

Comment:
This test is deliberately violating the signature of `Fn.join`. I don't think it is valid much anymore.

Reply:
stringified/arrayified the tokens so "as any" is no longer needed